### PR TITLE
*: clean code.

### PR DESCRIPTION
**Pull request by hanfei1991**
remove useless code.
@zimulala @shenli @coocood @tiancaiamao @XuHuaiYu PTAL

### CODE REVIEW

File: executor/adapter.go [hunk 1]
```diff
@@ -131,15 +131,8 @@ func (a *statement) Exec(ctx context.Context) (ast.RecordSet, error) {
 		}
 	}
 
-	fs := e.Fields()
```
Comment (shenli) [inline executor/adapter.go]: Is there any stmt still use result fields?

Comment (shenli) [inline executor/adapter.go]: Can we remove ResultField structure?

Comment (hanfei1991) [inline executor/adapter.go]: The "Fields" is no more used.

**Comment by zimulala** [comment]
Fix CI

**Comment by coocood** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**